src/Menu/Menu.test.js

```diff
@@ -157,7 +187,9 @@ describe('<Menu />', () => {
                 <MemoryRouter>
                     <Menu>
                         <MenuList>
-                            <MenuItem link='/' linkProps={{ 'data-sample': 'Sample' }} />
+                            <MenuItem urlProps={{ 'data-sample': 'Sample' }}>
+                                <Link to='/' />
+                            </MenuItem>
                         </MenuList>
```

Comment:
Are this and the next test now testing the same thing?

Reply:
They are not actually. One is testing using the React-Router `<Link>` where the other is testing using the internal creation of the `<a>` tag.